views/data.hbs

```diff
@@ -31,7 +31,6 @@
         <a id="close-mobile">&times;</a>
         <div class="content"></div>
         This website is optimized for desktop usage.
-        </div>
```

Comment:
In this case you may have not removed the right closing div. What I suspect is that your IDE helps you and put the closing div automatically when you close the opening tag on line 31. I suspect the text on line 32 should be embraced by the div?